types/mirrorx/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+	"files": [
+        "index.d.ts",
+        "mirrorx-tests.ts"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
Please set `strictNullChecks` to `true` on all new packages.

Reply:
ok.

Reply:
Done.